sources/packages/backend/libs/src-sql/sfas-restrictions/Bulk-insert-sfas-mapped-restrictions.sql

```diff
@@ -38,6 +38,22 @@ FROM
 WHERE
   sfas_individuals.student_id IS NOT NULL
   AND (
-    student_restrictions.is_active = false
-    OR student_restrictions.restriction_id IS NULL
+    student_restrictions.restriction_id IS NULL
```

Comment:
Is it not possible to add the `is_active` condition to the `LEFT JOIN` as below or something similar?
```sql
LEFT JOIN sims.student_restrictions student_restrictions
   ON student_restrictions.student_id = sfas_individuals.student_id
      AND student_restrictions.is_active = true
```

Reply:
Thanks for suggesting it @andrewsignori-aot . Simplifies the logic a lot.